webpack.config.ts

```diff
@@ -11,13 +11,15 @@ export type ConfigOptions = {
   context?: string
   sourceMap?: 'source-map' | 'eval-source-map' | false
   transpileOnly?: boolean
+  rootDir: string
 }
 
 export function getClientConfig({
   mode,
   context,
   sourceMap,
   transpileOnly,
+  rootDir,
```

Comment:
Perhaps default its value (and make it optional), such that we don't have to pass it in most of the time?

```suggestion
  rootDir = __dirname,
```